cassandra-diagnostics-reporter-influx/src/main/java/io/smartcat/cassandra/diagnostics/reporter/InfluxReporter.java

```diff
@@ -106,10 +106,12 @@ public void report(Query queryReport) {
         logger.debug("Sending Query: {}", queryReport.toString());
         try {
             influx.write(dbName, retentionPolicy,
-                    Point.measurement(measurementName).time(queryReport.startTimeInMilliseconds, TimeUnit.MILLISECONDS)
+                    Point.measurement(measurementName)
+                            .time(queryReport.getStartTimeInMilliseconds(), TimeUnit.MILLISECONDS)
                             .tag("host", hostname).tag("id", UUID.randomUUID().toString())
-                            .addField("client", queryReport.clientAddress).addField("statement", queryReport.statement)
-                            .addField("value", queryReport.executionTimeInMilliseconds).build());
+                            .addField("client", queryReport.getClientAddress())
+                            .addField("statement", queryReport.getStatement())
```

Comment:
We need statement as stag instead of field since we want to query on type of statment. I have changed this on master of old project so this can be good point to change it here as well.